packages/ui/src/stories/foundation/typography.stories.tsx

```diff
@@ -91,3 +91,48 @@ export const BodyFont = () => {
     </Table>
   );
 };
+
+/**
+ * Line height
+ */
+export const LineHeight = () => {
+  return (
+    <div>
+      <p className="mb-2">
+        The pre-Next version of GEL had a fall-back line height of 1.428571429 for things inside the body, the reasoning
+        for this can be found{' '}
+        <Link
+          href="https://stackoverflow.com/questions/19982651/why-does-bootstrap-set-the-line-height-property-to-1-428571429"
+          type="inline"
+        >
+          here
+        </Link>
+        . A typography type has been added and the leading style in tailwind (with 14px font) has been extended to
+        include this value for use to match the older version of GEL.
+      </p>
+      <Table>
+        <Table.Caption>GEL 'loose' line-height</Table.Caption>
+        <Table.Header>
+          <Table.HeaderRow>
+            <Table.HeaderCell>Tailwind Class</Table.HeaderCell>
+            <Table.HeaderCell>CSS Value</Table.HeaderCell>
+          </Table.HeaderRow>
+        </Table.Header>
+        <Table.Body>
+          <Table.Row key="typography-11">
+            <Table.Cell>
+              <span className="typography-body-11">typography-body-11</span>
+            </Table.Cell>
+            <Table.Cell>fontSize: 0.875rem; lineHeight: 1.428571429</Table.Cell>
+          </Table.Row>
```

Comment:
We may need to add all three options we declare for line height in our docs. We have a config in Tailwind for `normal`, `tight` and `loose`.